### PR TITLE
Add GitHub ISSUE_TEMPLATE files

### PR DESCRIPTION
**Pull request by cliffordfajardo**
## What is the change?
Add `bug_report.yml` & `config.yml` file to enable Github's form based issue template
- https://youtu.be/qQE1BUkf2-s?t=23

## Motivation
- encourage's bug reporter's to put more care into their bug report before submission
- this may help maintainer's receive more detailed & higher quality bug report's
- adds helpful tips for user's during the process of creating a bug/issue report

## Demo of Change
this PR is similar to this one I created here for another repo recently 
- https://github.com/antvis/G6/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🤔 Feature Requests & Questions 
+    url: https://github.com/sergiodxa/remix-i18next/discussions
+    about: Please ask and answer questions here.
+  - name: 💬 Remix Discord Channel 
+    url: https://rmx.as/discord
+    about: Interact with other people using Remix 📀
+  - name: 💬 New Updates (Twitter)
+    url: https://twitter.com/remix_run
```
Comment (sergiodxa) [inline .github/ISSUE_TEMPLATE/config.yml:10]: Remix account will never tweet updates of this library, the lib was made by me not them, and I also don’t tweet about new versions, you should follow releases.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    about: Stay up to date with Remix news on twitter
```
Comment (sergiodxa) [inline .github/ISSUE_TEMPLATE/config.yml:11]: Same thing, they will not upload videos for this package

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (cliffordfajardo) [reply .github/ISSUE_TEMPLATE/config.yml:10]: Ahh I can see how this might confuse some folks - good point

Comment (cliffordfajardo) [reply .github/ISSUE_TEMPLATE/config.yml]: Updated 👌 
- https://github.com/sergiodxa/remix-i18next/pull/41/commits/423a2323dc312918949bd3ef6b1e58da242f5225

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+name: '🐛 Bug report'
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in Remix Auth (https://github.com/sergiodxa/remix-i18next).
```
Comment (sergiodxa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]:
```suggestion
        This issue tracker is for reporting bugs found in Remix i18next (https://github.com/sergiodxa/remix-i18next).
```

**Review by sergiodxa — APPROVED** [review]

**Comment by cliffordfajardo** [comment]
Thanks for taking a look - 🚀🙏

**Pull request merged**